frontend/src/components/dashboard/subdomain/SubdomainInfoTooltip.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+  useButton,
+  useOverlay,
+  useModal,
+  useDialog,
+  FocusScope,
+  OverlayContainer,
+  useOverlayPosition,
+  AriaOverlayProps,
+  mergeProps,
+} from "react-aria";
+import {
+  forwardRef,
+  HTMLAttributes,
+  ReactNode,
+  RefObject,
+  useRef,
+} from "react";
+import { useOverlayTriggerState } from "react-stately";
+import { CloseIcon, InfoIcon } from "../../Icons";
+import { useMinViewportWidth } from "../../../hooks/mediaQuery";
+import { useL10n } from "../../../hooks/l10n";
+import styles from "./SubdomainInfoTooltip.module.scss";
+import { Localized } from "../../Localized";
+import { InfoModal } from "../../InfoModal";
+
+/**
+ * Shows the user more info on how to use their Relay custom domain mask.
+ */
+export const SubdomainInfoTooltip = () => {
+  const isLargeScreen = useMinViewportWidth("md");
+
+  return <>{isLargeScreen ? <ExplainerTrigger /> : <MobileExplainerModal />}</>;
+};
+
+const MobileExplainerModal = () => {
+  const modalState = useOverlayTriggerState({});
+  const l10n = useL10n();
+
+  const subdomainTooltipButton = (
+    <button
+      className={styles["info-icon"]}
+      onClick={() => {
+        modalState.open();
+      }}
+    >
+      <InfoIcon
+        alt={l10n.getString("tooltip-email-domain-explanation-title")}
+        width={18}
+        height={18}
+      />
+    </button>
+  );
+
+  const subdomainExplanationBody = (
+    <>
+      <p>{l10n.getString("tooltip-email-domain-explanation-part-one")}</p>
+      <br />
```

Comment:
question: Is a `<br />` necessary? I would recommend adding margin/padding to the `<p>` instead. (Looking through the codebase and @lloan @codemist @Vinnl @groovecoder have added `<br>` tags too too 😭 )